app/src/helpers/helpers.ts

```diff
@@ -262,6 +262,16 @@ export function openExternal(
   return shell.openExternal(url, options);
 }
 
+// Copy-pastaed as unable to get imports to work in preload.
+// If modifying, update also app/src/preload.ts
+export function isWayland(): boolean {
+  return (
+    isLinux() &&
+    (Boolean(process.env.WAYLAND_DISPLAY) ||
```

Comment:
For the record `Boolean` or `boolean` on a string only returns its truthiness (i.e., is it not blank, null, nor undefined).

```JavaScript
> Boolean('true')
true
> Boolean('false')
true
```

We have written a `parseBoolean` function in `src/utils/parseUtils.ts` that should probably be copy and pasted here for use in the app.

Reply:
I think in this case `Boolean` is okay, as the `WAYLAND_DISPLAY` environment variable does not contain a `"true"/"false"` value, but rather the name of a display. So what we want to check here is just whether it has been set to some value (not null or empty) or not.